sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -259,8 +264,16 @@ e6000sw_probe(device_t dev)
 		 */
 		resource_int_value(device_get_name(sc->dev),
 		    device_get_unit(sc->dev), "is8190", &is_6190);
+
+	if (is_6190 !=0 && is_6190x != 0)
+		if (bootverbose)
+			device_printf(dev, "Cannot configure conflicting variants\n");
+		return (ENXIO);
```

Comment:
Given the lack of curly braces amounts to an unconditional `return (ENXIO)` I don't see how this could possibly have been tested.